Package.swift

```diff
@@ -88,5 +89,24 @@ let package = Package(
 				.copy("Resources"),
 			]
 		),
+        
+        // MARK: KotlinStandAlones
+        
+        .target(
+            name: "KotlinStandAlone",
```

Comment:
Add this new `KotlinStandAlone` target as a dependency to the `LexiconGenerators` target. Then when you build that target you build all generators.